src/app/components/client/ProgressCard.tsx

```diff
@@ -0,0 +1,110 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ "use client";
+
+ import { CSSProperties, useState } from "react";
+ import { QuestionMarkCircle } from "../server/Icons";
+ import styles from "./ProgressCard.module.scss";
+ import ExploringLaptopPlus from "./assets/exploring-laptop-check.png";
+ import ExploringLaptopMinus from "./assets/exploring-laptop-minus.png";
+ import SparklingCheck from "./assets/sparkling-check.png";
+ import Image from "next/image";
```

Comment:
Formatting: Most of the lines in this `ProgressCard` have an additional whitespace that should be removed.

```suggestion
import Image from "next/image";
import { CSSProperties, useState } from "react";
import { QuestionMarkCircle } from "../server/Icons";
import styles from "./ProgressCard.module.scss";
import ExploringLaptopPlus from "./assets/exploring-laptop-check.png";
import ExploringLaptopMinus from "./assets/exploring-laptop-minus.png";
import SparklingCheck from "./assets/sparkling-check.png";
```